e2e/terraform/nomad-acls.tf

```diff
@@ -36,8 +36,8 @@ locals {
 cat <<ENV | sudo tee -a /root/.bashrc
 export NOMAD_ADDR=https://localhost:4646
 export NOMAD_SKIP_VERIFY=true
-export NOMAD_CLIENT_CERT=/etc/nomad.d/tls/agent.crt
-export NOMAD_CLIENT_KEY=/etc/nomad.d/tls/agent.key
+export NOMAD_CLIENT_CERT="${path.module}/etc/nomad.d/tls/agent.crt"
+export NOMAD_CLIENT_KEY="${path.module}/etc/nomad.d/tls/agent.key"
```

Comment:
Isn't this path the path on the _server_, not the terraform environment? I don't think we want to change this.